### PR TITLE
Updates to getting started guide

### PR DESCRIPTION
**Pull request by alindfor**
Added information about ubuntu and CentOS installation

This PR addresses #

Proposed changes include:
* describe the functionality changes
* describe new or updated tests
* describe changes to the documentation

### CODE REVIEW

File: doc/user-guide/Getting-started.md [hunk 1]
```diff
@@ -7,23 +7,53 @@ You can either compile everything from the source code or install binaries from
 
 Check out our tutorial [How to build MongooseIM from source code](How-to-build.md) for an introduction to compiling, building and testing MongooseIM.
 
-### Install from DMG
+### Install from package
+
+Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) page of Erlang Solution website, and download the version of MongooseIM you want. The following sections describe the installation process for different operating systems.
+
+#### Mac Os
```
Comment (fenek) [inline doc/user-guide/Getting-started.md]: It's either `macOS` or `Mac OS X` or `OS X`. Or the safest choice: `Mac`.

File: doc/user-guide/Getting-started.md [hunk 2]
```diff
@@ -40,7 +70,7 @@ mongooseimctl register user domain password
 
 ## Connecting with an XMPP client
 
-### Adium
+### Adium (Mac OS)
```
Comment (fenek) [inline doc/user-guide/Getting-started.md]: See the previous comment about "Mac Os"

File: doc/user-guide/Getting-started.md [hunk 3]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
 
 Registered users wishing to add an existing account to Adium should enter the MongooseIM server’s IP address in the **Connect Server** field on the **Options** tab.
 
+### Pidgin (Ubuntu & CentOS)
+
+1. 	Launch Pidgin.
```
Comment (fenek) [inline doc/user-guide/Getting-started.md]: Indentation inconsistent with other numbered lists in the file.

File: doc/user-guide/Getting-started.md [hunk 4]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
...
+2. 	Click the Add button to configure your account.
+3. 	In the **Basic** tab choose **XMPP** as protocol,
+		then enter Username, Domain and Password and click **Add**.
```
Comment (fenek) [inline doc/user-guide/Getting-started.md]: We usually stick to a "one sentence per line" convention in docs (without characters per line limit), so no need to break it into several lines.

File: doc/user-guide/Getting-started.md [hunk 5]
```diff
@@ -7,23 +7,53 @@ You can either compile everything from the source code or install binaries from
 
 Check out our tutorial [How to build MongooseIM from source code](How-to-build.md) for an introduction to compiling, building and testing MongooseIM.
 
-### Install from DMG
+### Install from package
+
+Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) page of Erlang Solution website, and download the version of MongooseIM you want. The following sections describe the installation process for different operating systems.
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) section of the Erlang Solution website, and choose the version of MongooseIM you want. The following sections describe the installation process for different operating systems.

File: doc/user-guide/Getting-started.md [hunk 6]
```diff
@@ -7,23 +7,53 @@ You can either compile everything from the source code or install binaries from
...
+$ sudo rpm -i mongooseim_2.1.0-1~centos~7_amd64.rpm
+```
+
+## Running MongooseIM installed from package
+
+The following command will start the MongooseIM server:
+```bash
+$ mongooseimctl start
+```
+The following command shows the status of a started MongooseIM server
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: The following command shows the status of a started MongooseIM server:

File: doc/user-guide/Getting-started.md [hunk 7]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
 
 Registered users wishing to add an existing account to Adium should enter the MongooseIM server’s IP address in the **Connect Server** field on the **Options** tab.
 
+### Pidgin (Ubuntu & CentOS)
+
+1. 	Launch Pidgin.
+2. 	Click the Add button to configure your account.
+3. 	In the **Basic** tab choose **XMPP** as protocol,
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: In the **Basic** tab choose **XMPP** as the protocol

File: doc/user-guide/Getting-started.md (hunk 4 above)
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: then enter the Username, Domain and Password and click **Add**.

File: doc/user-guide/Getting-started.md [hunk 8]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
...
+		then enter Username, Domain and Password and click **Add**.
+		After Registration the user will connect automatically.
+
+### Gajim (Ubuntu & CentOS)
+
+The following steps assumes that you have already registered a user on the MongooseIM server, see section **Registering a user** above.
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: The following steps assume that you have already registered a user on the MongooseIM server (see section **Registering a user** above).

File: doc/user-guide/Getting-started.md [hunk 9]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
...
+1. Launch Gajim. Ignore window about Plugin updates.
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: Launch Gajim. Ignore the window with Plugin updates.

File: doc/user-guide/Getting-started.md [hunk 10]
```diff
@@ -55,6 +85,22 @@ After registration, the user will connect automatically.
...
+2. Go to Edit -> Accounts.
+3. Click Add in the left part of the window and select **I already have an account I want to use**, click Forward
+4. Enter user, domain and password for the already registered account, click Forward and then Finish.
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]: Enter the user, domain and password for the already registered account, click Forward and then Finish.

**Pull request merged**